### PR TITLE
Change fontstyle of Labels and order of Datepicker

### PR DESCRIPTION
**Pull request by JoyLubega**
Resolve https://github.com/opencollective/opencollective/issues/4900

# Description

- [x] Change order

- [x] Change label style

# Screenshots

Screenshot 2022-01-25 at 11 42 03 PM

### CODE REVIEW

File: components/budget/filters/PeriodFilter.js [hunk 1]
```diff
@@ -154,8 +154,52 @@ const PeriodFilter = ({ onChange, value, inputId, minDate, ...props }) => {
               interval={tmpDateInterval}
             />
           </Box>
+          <StyledInputField
+            name="timezoneType"
+            mt="12px"
+            labelFontSize="13px"
```
Comment (SudharakaP) [inline components/budget/filters/PeriodFilter.js:160]: According to design; 

```suggestion
            labelFontSize="16px"
```

File: components/budget/filters/PeriodFilter.js [hunk 2]
```diff
@@ -154,8 +154,52 @@ const PeriodFilter = ({ onChange, value, inputId, minDate, ...props }) => {
...
+                </StyledTooltip>
+              </Flex>
+            }
+          >
+            {inputProps => (
+              <StyledButtonSet
+                {...inputProps}
+                size="tiny"
+                items={['local', 'UTC']}
+                buttonProps={{ p: 1, fontSize: '11px' }}
```
Comment (SudharakaP) [inline components/budget/filters/PeriodFilter.js:182]: According to design,

```suggestion
                buttonProps={{ p: 1, fontSize: '13px', fontWeight: 400 }}
```

**Review by SudharakaP — CHANGES_REQUESTED** [review]
@JoyLubega : Thanks much for the contributions. 😄 

Some minor feedback, double checking the design. Also note that the `Reset` and `Apply` button font size should be `14px` which means we have to change the `buttonSize` prop to `medium` I think. 😉

**Comment by JoyLubega** [comment]
Screenshot 2022-01-26 at 12 39 13 PM

@SudharakaP  I made the changes, 14px for buttons make them way too bigger.

**Comment by Betree** [comment]
> > @SudharakaP I made the changes, 14px for buttons make them way too bigger.
> 
> @Betree : Do you think this is too big? For me it looks okay. thinking

I haven't inspected the styles, but the screenshot seems to match the design

**Review by SudharakaP — APPROVED** [review]

**Pull request merged**